### PR TITLE
Revisions: don't create autosave matching post

### PR DESCRIPTION
**Pull request by adamsilverstein**
Trac ticket: https://core.trac.wordpress.org/ticket/49532

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php [hunk 1]
```diff
@@ -360,35 +360,34 @@ public function create_post_autosave( $post_data ) {
 			return $post;
 		}
 
+		// Only create an autosave when the it is different than the saved post.
```
Comment (costdev) [inline src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php]:
```suggestion
		// Only create an autosave when it is different to the saved post.
```

Comment (adamsilverstein) [reply src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php]: thanks for the correction! (think "to" should still be "than" though)

Comment (costdev) [reply src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php]: > The one big difference between these terms is that 'different from' and 'different to' are standard in British English. 'Different than', meanwhile, is primarily used in American English.

That explains it! Happy to close this one as resolved, thanks @adamsilverstein!

Comment (adamsilverstein) [reply src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php]: Oh, I had no idea the British English usage was different. Good to know, thanks! 

I'll accept your suggestion to fix the extraneous "the", then revert the language change bit to keep the American usage which is I believe standard for core.

**Comment by adamsilverstein** [comment]
merged in https://core.trac.wordpress.org/changeset/55154

**Pull request closed without merging**